macOS/DuckDuckGoVPN/DuckDuckGoVPNAppDelegate.swift

```diff
@@ -373,7 +373,7 @@ final class DuckDuckGoVPNAppDelegate: NSObject, NSApplicationDelegate {
     }()
 
     private func statusViewSubmenu() -> [StatusBarMenu.MenuItem] {
-        let appLauncher = AppLauncher(appBundleURL: Bundle.main.bundleURL)
+        let appLauncher = AppLauncher()
```

Comment:
This code is in the VPN menu app, and the launcher was targetting the VPN menu app instead of the main app.  The default parameter-less initializer points to the main app instead.